packages/itwinui-react/src/core/Table/filters/DateRangeFilter/DateRangeFilter.tsx

```diff
@@ -43,6 +44,7 @@ export type DateRangeFilterProps<T extends Record<string, unknown>> =
     parseInput?: (text: string) => Date;
     placeholder?: string;
     translatedLabels?: DateRangeTranslation & FilterButtonBarTranslation;
+    localizedNames?: DatePickerLocalizedNames;
```

Comment:
maybe this could be a new key inside the existing `translatedLabels` instead of a new prop?

e.g. `translatedLabels.datePicker`

Reply:
Good point! I removed the prop and added as a new key